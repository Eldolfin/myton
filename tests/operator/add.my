print(123 + 456) # 579

print("str" + "ing") # string

print("str" + 123) # str123

print(123 + "str") # 123str

print("123" + "456") # 579

print("123" + 456) # 579

print(123 + "456") # 579
