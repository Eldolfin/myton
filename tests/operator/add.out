579
string
str123
123str
579
579
579
